src/main/java/soot/jimple/toolkits/typing/fast/TypeUtils.java

```diff
@@ -64,7 +64,15 @@ public static int getValueBitSize(Type type) {
     if (type instanceof DoubleType) {
       return 64;
     }
-
+    if (type instanceof Integer127Type) {
+      return 7;
```

Comment:
should be 8 bit for signed -127 <-> +128

Reply:
The Integer127Type prints "[0..127] in it's `toString` method, therefore I thought it is a 7bit data type. But from your comment I assume that the relevant type is the one returned by it's `getDefaultFinalType` method (which is `ByteType`).
I will correct this.